docker-compose.yml

```diff
@@ -4,11 +4,11 @@ services:
   db:
     image: postgres:14
     environment: &common-environment
-      POSTGRES_USER: ${DATABASE_USER:-postgres}
-      POSTGRES_PASSWORD: ${DATABASE_PASSWORD:-postgres}
-      POSTGRES_DB: ${DATABASE_NAME:-dive}
-      POSTGRES_HOSTNAME: ${DATABASE_HOSTNAME:-db}
-      POSTGRES_PORT: ${DATABASE_PORT:-5432}
+      POSTGRES_USER: ${POSTGRES_USER:-postgres}
```

Comment:
The changes need to reflect in the django app code as well. Please add that as well.

Reply:
My bad! I see the changes have been made.